lms/djangoapps/grades/signals/signals.py

```diff
@@ -81,3 +81,23 @@
         'subsection_grade',  # SubsectionGrade object
     ]
 )
+
+# Signal that indicates that a user's score for a subsection has been overridden.
+# This signal is generated when an admin changes a user's exam attempt state to
```

Comment:
Tiny nit: it's not always an admin, the Software Secure callbacks do it sometimes too. I don't think the "who" of it matters, it's just "the exam attempt changes state"